frontend/src/components/Tutorial/Steps/students.js

```diff
@@ -0,0 +1,35 @@
+import agent from 'services/agent';
+
+const steps = [
+  {
+    placement: 'center',
+    target: 'body',
+    title: 'Gérer mes étudiants',
+    content: "Cette page vous permet d'ajouter et de visualiser les étudiants.",
+  },
+  {
+    placement: 'auto',
+    target: '#new-student-button',
+    content: "Pour enregistrer un nouvel étudiant, cliquer sur ce bouton et renseigner les informations de l'étudiant.",
+  },
+  {
+    placement: 'auto',
+    target: '#students-table',
+    shouldSkip: () => agent.Patient.get().then(students => students.length > 0),
+    content: "Les étudiants enregistrés apparaîtront ici, vous pourrez les supprimer en cas d'erreur ou modifier leurs informations.",
+  },
+  {
+    placement: 'auto',
+    target: '#students-table',
+    shouldSkip: () => agent.Patient.get().then(students => students.length === 0),
+    content: 'Ce tableau liste tous les étudiants que vous avez enregistré, le statut de leur dossier et le nombre total de séances que vous avez déclaré pour lui/elle.',
+  },
+  {
+    placement: 'auto',
```

Comment:
I tried to use 'top-end' and 'right-start' placement here but it doesn't seem to work :(

Reply:
it's working for me, we can look together